### DISCUSSION TITLE
Not all entries are displayed in the "Not installed" list

### DISCUSSION THREAD

**Discussion by andrey-helldar** [yhype/yhype-feedback#25] [Q&A] [answered]
Not all entries are displayed in the "Not installed" list. In my case, one entry is missing - "Volunteers CRM". This is my project, available at the link: https://github.com/volunteers-crm

Clicking on the "Sync accounts data" button has no effect. I tried the same after solving the problem in the neighboring discussion: [yhype/discussions#24](https://github.com/orgs/yhype/discussions/24)

**Comment by antonkomarev** [comment, marked as answer]
@andrey-helldar Are you a public member of the organisation? GitHub does not allow to get org info of the private members.

https://yhype.me/faq

**Reply by andrey-helldar** [reply]
Interesting. I haven't looked in that section in any of my repositories. And yes, I was listed there as "private".

It's works! Thank you!

![image](https://github.com/yhype/yhype-feedback/assets/10347617/60ac5ab3-95f9-4e92-accc-5ce6759c01f8)
![image](https://github.com/yhype/yhype-feedback/assets/10347617/5a0c6e09-0cc4-4d7d-a938-0357cafd04f8)
